be/issue-tracker-be/src/main/java/com/codesquad/issuetracker/user/UserDummyData.java

```diff
@@ -0,0 +1,42 @@
+package com.codesquad.issuetracker.user;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public class UserDummyData {
+    private UserDummyData() {
+
+    }
+
+    public static Set<UserResponse> usersResponse() {
+        return new HashSet<>(Arrays.asList(
+                UserResponse.builder()
+                    .id(1L)
+                    .name("hiro")
+                    .email("hiro@naver.com")
+                    .build(),
+                UserResponse.builder()
+                    .id(1L)
+                    .name("freddie")
+                    .email("freddie@naver.com")
+                    .build()
```

Comment:
`userFreddie()`와 `userHiro()` 사용하면 되는 부분 같습니다
```suggestion
                userFreddie(),
                userHiro()```